lib/binaries/CLI.js

```diff
@@ -42,6 +42,7 @@ commander.version(pkg.version)
   .option('--env <environment_name>', 'specify which set of environment variables from ecosystem file must be injected')
   .option('-a --update-env', 'force an update of the environment with restart/reload (-a <=> apply)')
   .option('-f --force', 'force actions')
+  .option('--filter_env [envs]', 'filter outgoing global values by prefix', function(v, m) { m.push(v); return m;}, [])
```

Comment:
We use `includes` in the `filterEnv` function and it checks if the string contains passed substring. So we filter outgoing values by the presence of the provided value in the key.

Reply:
I hope that it will be in the release soon....

Reply:
@inerc I hope so, my pm2 crashes because of the env variables overflow. Can you change the description above? What about: `'filter out outgoing global values that contain provided strings'`?

Reply:
Yes, it will be better. I have made changes.